sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -179,6 +189,25 @@ export class StudentAESTController extends BaseController {
     );
   }
 
+  /**
+   * Search students based on the search criteria.
+   * @param searchCriteria criteria to be used in the search.
+   * @returns searched student details.
+   */
+  @Get("search")
+  async searchStudents(
+    @Query() searchCriteria: AESTStudentSearchAPIInDTO,
+  ): Promise<SearchStudentAPIOutDTO[]> {
+    const searchStudentApplications =
+      await this.studentService.searchStudentApplication(searchCriteria);
+    return searchStudentApplications.map((eachStudent: Student) => ({
+      id: eachStudent.id,
+      firstName: eachStudent.user.firstName,
+      lastName: eachStudent.user.lastName,
+      birthDate: getISODateOnlyString(eachStudent.birthDate),
```

Comment:
As per figma, all the dates at UI must be in this format I believe. From here we are sending the ISO format. Are we expecting the client side to transform it again?
![image](https://user-images.githubusercontent.com/54600590/170783433-c0e9ff08-2ea6-410c-bdaa-263271772212.png)

Reply:
As we talked about in the past it would be best to API return the date itself (and as a string) when we have the date-only.
And yes, the Vue is supposed to do the transformation, it was not, I forgot, and I just added 😄 
![image](https://user-images.githubusercontent.com/61259237/170784995-71091a4a-369c-4eb4-b3bb-6fd5323a0bdd.png)

Reply:
The comma is now removed from the dates formatted with `dateOnlyLongString`.

Reply:
thanks for doing that @andrewsignori-aot 👍